microsoft-edge/webview2/how-to/set-preview-channel.md

```diff
@@ -6,259 +6,379 @@ ms.author: msedgedevrel
+The release channels can similarly be set through the API, registry key, environment variable, or group policy.
 
-To make your application use a Microsoft Edge preview channel by using a registry override that changes the release channel preference by changing the order of searching for a channel:
+
+##### [API](#tab/api)
+
+By default, the loader considers all of the release channels.  Set `ReleaseChannels` to a mask of the channels you would like the loader to search for.
+
+.NET:
+* [CoreWebView2EnvironmentOptions.ReleaseChannels Property](/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions.releasechannels)
+* [CoreWebView2ReleaseChannels Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2releasechannels)
+
+WinRT:
+* [CoreWebView2EnvironmentOptions.ReleaseChannels Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions#releasechannels)
+* [CoreWebView2ChannelSearchKind Enum](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2releasechannels)
+
+Win32:
+* [ICoreWebView2EnvironmentOptions7::get_ReleaseChannels](/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions7#get_releasechannels)
+* [ICoreWebView2EnvironmentOptions7::put_ReleaseChannels](/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions7#put_releasechannels)
+* [COREWEBVIEW2_RELEASE_CHANNELS enum](/microsoft-edge/webview2/reference/win32/webview2-idl#corewebview2_release_channels)
+
+
+##### [Registry key](#tab/registry-key)
 
 1. Open a PowerShell terminal or PowerShell-enabled command prompt.
 
 1. Modify and then run the following command:
 
-   `REG ADD HKLM\Software\Policies\Microsoft\Edge\WebView2\ReleaseChannelPreference /v WebView2APISample.exe /t REG_SZ /d "1"`
+   ```reg
+   REG ADD <HKLM/HKCU>\Software\Policies\Microsoft\Edge\WebView2\ReleaseChannels /v WebView2APISample.exe /t REG_SZ /d "0,1"
+   ```
 
-   Replace `WebView2APISample.exe` with the file name of your own app executable or the Application User Model ID. Using a wildcard (*) as the value name will apply the override to _all_ WebView2 apps on the machine and can result in unexpected behavior.
+Replace `WebView2APISample.exe` with your own app executable name or the application user model ID.  Using a wildcard (*) as the value name will apply the override to _all_ WebView2 apps on the machine, and can result in unexpected behavior.
 
-#### Resuming using the default, WebView2 Evergreen Runtime
 
-To delete the `ReleaseChannelPreference` registry override, run the command:
+##### [Environment variable](#tab/environment-variable)
 
-`REG DELETE HKLM\Software\Policies\Microsoft\Edge\WebView2\ReleaseChannelPreference /f`
+Name: `WEBVIEW2_RELEASE_CHANNELS`  
+Value: `<string of desired channels (ex: 0,1)>`
 
+The environment variable if set globally is applied to all apps that use WebView2 on the machine, unlike the registry key, which can be set per app.
```

Comment:
check wording here too; sync w/ elsewhere in this .md file.

Reply:
Rendered articles for review: 
* **Test upcoming APIs and features** (update to use new API)
   * https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/set-preview-channel?branch=pr-en-us-3053
* **Prerelease testing** (new article)
   * https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/prerelease-testing?branch=pr-en-us-3053
* **Self-hosting** (new article)
   * https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/self-hosting?branch=pr-en-us-3053

For "Outdated" comments, to see latest Markdown/commits, use fresh Files Changed tab:
https://github.com/MicrosoftDocs/edge-developer/pull/3053/files

Reply:
I wouldn't mind per-app as well.

Reply:
I might: First rewrite the two spots as "per-app", and then enter two fresh comments to possibly sync the wording in both spots, then Resolve present thread.

Reply:
entered at both "per-app" spots:
```
<!-- todo: there are two "per-app" spots in this .md file.  Make them same?
The environment variable will be applied to all apps that use WebView2 on the machine, unlike the registry key, which can be set per-app.
The environment variable, if set globally, is applied to all apps that use WebView2 on the machine.  In contrast, the registry key can be set per-app.
-->
```